README.md

```diff
@@ -188,13 +194,17 @@ You can hover over an element while pressing the `CTRL` key to view its document
 
 ![](resources/images/lsp4intellij-hover.gif)
 
+#### Workspace Symbols
+Navigate to **Navigate->Symbol...** and enter the symbol name you want to search in the search box popup.
```

Comment:
```suggestion
Navigate to the **Navigate->Symbol...** and enter the name of the symbol you want to search in the search box that pops up.
```
What is **Navigate->Symbol...**? YDid we mean navigate to some place and click "Symbol"?

Reply:
Please refer the attachment below.

![navigate-symbol](https://user-images.githubusercontent.com/29032600/56257707-c37eed80-60ea-11e9-9263-459e1a7fac96.gif)